README.md

```diff
@@ -45,6 +45,31 @@ about them:
 Similarly, you may need to `npm link path/to/your/js/sdk` in your `matrix-react-sdk`
 directory.
 
+Example of installation and link for vector:
+
+```
+mkdir ~/matrix
+cd ~/matrix
+git clone https://github.com/vector-im/vector-web.git
+cd ~/vector-web
+git checkout develop
```

Comment:
Ok I understand but I did not played with matrix-js-sdk yet. And it works for me. Maybe I missed something?

I can reorder to first clone an build matrix-react-sdk and then clone and build vector-web and then link both.